framework_tvb/tvb/core/portlets/__init__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
This module should also be marked deprecated or removed directly as part of small 'cleanup' PR

Reply:
Ok, I will remove the portlets module in a separate PR after this one.